### PR TITLE
Regenerator runtime missing

### PR DESCRIPTION
**Pull request by victorcuena**
screen shot 2019-01-25 at 15 26 43

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ src/locales
 *.pid
 main.js
 eslint.report.html
+lib
```
Comment (rnowosielski) [inline .gitignore]: https://stackoverflow.com/questions/24942161/does-npm-ignore-files-listed-in-gitignore

Just make sure that the published package is actually will contain the lib folder. Last time somebody added lib to gitignore and the resulting package didn't have any code...

File: package.json [hunk 2]
```diff
@@ -61,6 +60,7 @@
     "jwt-decode": "^2.2.0",
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
+    "regenerator-runtime": "^0.12.1",
```
Comment (istanishev) [inline package.json]: Is that still necessary?

Comment (rnowosielski) [reply package.json]: It shouldn't be.

Comment (rnowosielski) [reply package.json]: I think it got there accidentally - will remove it.

**Pull request merged**